open_spiel/scripts/global_variables.sh

```diff
@@ -101,3 +101,11 @@ export BUILD_WITH_ORTOOLS="${BUILD_WITH_ORTOOLS:-"OFF"}"
 # https://github.com/google/or-tools/releases/download/v8.0/or-tools_debian-10_v8.0.8283.tar.gz
 # https://github.com/google/or-tools/releases/download/v8.0/or-tools_MacOsX-10.15.7_v8.0.8283.tar.gz
 export BUILD_WITH_ORTOOLS_DOWNLOAD_URL="${BUILD_WITH_ORTOOLS_DOWNLOAD_URL:-"https://github.com/google/or-tools/releases/download/v8.0/or-tools_ubuntu-18.04_v8.0.8283.tar.gz"}"
+
+# Used to determine whether to include the Python ML frameworks in the tests.
```

Comment:
If someone uses a fresh python install and wants to, I don't know, run AZ MCTS with pytorch, these won't be installed. I think it might be a good idea to update the documentation to mention this (something like "put ON instead of AUTO to install these extra deps")

Reply:
Thanks I will update the docs. However, as stated above, even setting these to ON will not install the frameworks. The user has to do that now.

Reply:
Ah, right... the ON install is done in travis script, I thought that was install.sh, my bad.